src/app/pages/apps-old/select-pool-dialog/select-pool-dialog.component.ts

```diff
@@ -98,8 +98,10 @@ export class SelectPoolDialogComponent implements OnInit {
           }));
           this.pools$ = of(poolOptions);
 
+          this.dialogService.closeAllDialogs();
+
           if (!pools.length) {
-            this.showNoPoolsWarning();
+            setTimeout(() => this.showNoPoolsWarning(), 100);
```

Comment:
This settimeout is also not needed

Reply:
@RehanY147 - there is problem with subscription here, because if we `closeAllDialogs` - we loose ref, I think so.